Lib/glyphsLib/util.py

```diff
@@ -68,3 +109,104 @@ def clear_data(data):
                 i += 1
         return data
     return True
+
+
+def fetch_all_glyphs(paths=()):
+    glyphs = {}
+    for fileobj in paths:
+        for glyph in etree.fromstring(fileobj).findall("glyph"):
```

Comment:
Yes, that was dumb. It was called filename before which was confusing because it also wasn't a filename.